strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/src/test/java/org/carlspring/strongbox/cron/jobs/RegenerateNugetChecksumCronJobTestIT.java

```diff
@@ -36,343 +41,277 @@
 
     @Override
     @BeforeEach
     public void init(TestInfo testInfo)
             throws Exception
     {
         super.init(testInfo);
-
-        createStorage(STORAGE1);
-
-        createRepository(STORAGE1, REPOSITORY_RELEASES, RepositoryPolicyEnum.RELEASE.getPolicy());
-
-        //Create released nuget package in the repository rnccj-releases (storage1)
-        generateNugetArtifact(getRepositoryBasedir(STORAGE1, REPOSITORY_RELEASES),
-                              "org.carlspring.strongbox.checksum-second", "1.0.0");
-
-        createRepository(STORAGE1, REPOSITORY_ALPHA, RepositoryPolicyEnum.SNAPSHOT.getPolicy());
-
-        //Create pre-released nuget package in the repository rnccj-alpha
-        generateAlphaNugetArtifact(getRepositoryBasedir(STORAGE1, REPOSITORY_ALPHA),
-                                   "org.carlspring.strongbox.checksum-one",
-                                   "1.0.1");
-
-        createStorage(STORAGE2);
-
-        createRepository(STORAGE2, REPOSITORY_RELEASES, RepositoryPolicyEnum.RELEASE.getPolicy());
-
-        //Create released nuget package in the repository rnccj-releases (storage2)
-        generateNugetArtifact(getRepositoryBasedir(STORAGE2, REPOSITORY_RELEASES),
-                              "org.carlspring.strongbox.checksum-one",
-                              "1.0.0");
-    }
-
-    @AfterEach
-    public void removeRepositories() throws IOException
-    {
-        removeRepositories(getRepositoriesToClean());
-    }
-
-    private String getRepositoryBasedir(String storageId,
-                                        String repositoryId)
-    {
-        return Paths.get(propertiesBooter.getVaultDirectory() +
-                         "/storages/" + storageId + "/" + repositoryId).toAbsolutePath().toString();
     }
 
+    @ExtendWith({ RepositoryManagementTestExecutionListener.class,
+                  ArtifactManagementTestExecutionListener.class })
     @Test
-    public void testRegenerateNugetArtifactChecksum()
+    public void testRegenerateNugetArtifactChecksum(@NugetRepository(repositoryId = REPOSITORY_RELEASES)
+                                                    Repository repository,
+                                                    @NugetTestArtifact(repositoryId = REPOSITORY_RELEASES,
+                                                                       id = "org.carlspring.strongbox.checksum-second",
+                                                                       versions = "1.0.0")
+                                                    Path artifactNupkgPath,
+                                                    @NugetTestArtifact(repositoryId = REPOSITORY_RELEASES,
+                                                                       id = "org.carlspring.strongbox.checksum-second",
+                                                                       versions = "1.0.0",
+                                                                       packaging = "nuspec")
```

Comment:
@ptirador it seems another confusion here. I found that you added this `packaging` parameter for `NugetArtifactGenerator` within [PR #1353](https://github.com/strongbox/strongbox/pull/1353), can you please remember why it was needed? The thing is that above `@NugetTestArtifact` annotation will generate `nuspec` file as well as the `nupkg`, so you can get it then with `artifactNupkgPath.resolveSbilling(artifactNupkgPath.getFileName().replace(".nupkg",".nuspec"))`. So probably we don't need this `packaging` attribute in annotation as well as the `packaging` parameter within `NugetPackageGenerator`.